storage/remote/codec.go

```diff
@@ -915,57 +621,6 @@ func LabelProtosToMetric(labelPairs []*prompb.Label) model.Metric {
 	return metric
 }
 
-// LabelProtosToLabels transforms prompb labels into labels. The labels builder
-// will be used to build the returned labels.
-func LabelProtosToLabels(b *labels.ScratchBuilder, labelPairs []prompb.Label) labels.Labels {
```

Comment:
This reverts https://github.com/prometheus/prometheus/pull/14316 cc @pracucci in some way, by removing this helper from this file.

This is actually accessible via `prompb.FromLabels` and `prompb.TimeSeries.ToLabels`. I think it makes things much cleaner and should work for downstream projects. Do you mind double checking if you are happy with this Marco? 🤗  (this PR is against remote-write-2.0 but we will merge this branch to main next week likely)

Reply:
Thanks for the ping Bartek. I'm good with this change, thanks.